app/controllers/samples_controller.rb

```diff
@@ -31,11 +31,10 @@ class SamplesController < ApplicationController
                    :dimensions, :all, :show_sample_names, :cli_user_instructions, :metadata_fields, :samples_going_public,
                    :search_suggestions, :stats, :upload, :validate_sample_files,].freeze
   OWNER_ACTIONS = [:raw_results_folder].freeze
+  TOKEN_AUTH_ACTIONS = [:create, :update, :bulk_upload, :bulk_upload_with_metadata].freeze
```

Comment:
I saw that you did not remove the `verify_authenticity_token` on this controller.

Reply:
Actually, I shouldn't have removed the other ones.